lib/Dialect/Torch/Transforms/AbstractInterpLibrary.cpp

```diff
@@ -5868,6 +5868,24 @@ StringRef mlir::torch::Torch::getAbstractInterpLibrary() {
 "    %3 = torch.prim.TupleConstruct %1, %0, %0 : !torch.list<int>, !torch.list<int>, !torch.list<int> -> !torch.tuple<list<int>, list<int>, list<int>>\n"
 "    return %3 : !torch.tuple<list<int>, list<int>, list<int>>\n"
 "  }\n"
+"  func.func @__torch__.torch.jit._shape_functions._batch_norm_with_update(%arg0: !torch.list<int>, %arg1: !torch.optional<list<int>>, %arg2: !torch.optional<list<int>>, %arg3: !torch.optional<list<int>>, %arg4: !torch.optional<list<int>>) -> !torch.tuple<list<int>, list<int>, list<int>, list<int>> {\n"
```

Comment:
I think that you forget to adding shape function of `_batch_norm_with_update` in `abstract_interp_lib_gen.py`? @vivekkhandelwal1

Reply:
But why the ci passed?

Reply:
> I think that you forget to adding shape function of `_batch_norm_with_update` in `abstract_interp_lib_gen.py`? @vivekkhandelwal1

No, this change comes from the upstream addition of this shape function.

Reply:
> But why the ci passed?

The same reason as above + we disabled the LTC build.